### PR TITLE
Add some content to Swagger from docs.msft.com

### PR DESCRIPTION
**Pull request by matthchr**
- This is in preparation for using the Swagger specification as the documentation
    authority that generates docs.msft.com documentation.

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by matthchr** [comment]
Swagger folks: Is there any reason I can't add tags or reviewers to this PR...?

**Comment by matthchr** [comment]
@itowlson @darylmsft @tamram - Please review

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (dsgouda) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: nit: for better documentation, consider inserting newline characters here (like so https://github.com/Azure/azure-rest-api-specs/blob/current/specification/sql/resource-manager/Microsoft.Sql/2015-05-01-preview/advisors.json#L54)

**Review by dsgouda — COMMENTED** [review]
Look like non-breaking changes. LGTM apart from a minor suggestion, will approve once ready to merge.

**Comment by dsgouda** [comment]
@erickson-doug since this is a doc related change, PTAL

**Comment by dsgouda** [comment]
@matthchr are you waiting on reviews from others before merging?

**Comment by matthchr** [comment]
@dsgouda Yes, please wait for some of my colleagues to review before merging. I will let you know when we're 100% ready to merge.

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >If not specified [](start = 200, length = 16)

If what is not specified?  Suggest something like: "If you do not specify a $filter clause including a startTime or endTime, these filters default to the start and end time of the last aggregation interval; that is..."

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >reccomended [](start = 718, length = 11)

typo 'recommended'

Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >remove any successfully added tasks [](start = 778, length = 35)

this is confusingly phrased.  I think we are saying "In a retry, it is most efficient to resubmit only tasks that failed to add, and to omit tasks that were successfully added on the first attempt."  Or something like that?

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >This [](start = 351, length = 4)

Suggest tweaking this to "Reactivation will fail..." as the antecedent of 'this' is unclear

Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >this [](start = 481, length = 4)

"reactivation", or "it" if you changed the previous sentence to refer to "reactivation"

Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: Or "this operation" would also work

---
In reply to: [133617601](https://github.com/Azure/azure-rest-api-specs/pull/1549#discussion_r133617601) [](ancestors = 133617601)

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >GetRemoteDesktop [](start = 298, length = 16)

Probably ought to change this to the MSDNified name e.g. "the 'connect to a compute node using Remote Desktop Protocol' API" or whatever they call it

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >cloud service configuration [](start = 175, length = 27)

"cloudServiceConfiguration" or "created in a cloud service configuration" (that is, if you are going to separate the words then don't say you're referring to the property)

Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >GetRemoteLoginSettings [](start = 277, length = 22)

is there a MSDN name for this?

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >tasks [](start = 60, length = 5)

task's with an apostrophe surely?

Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >tasks [](start = 98, length = 5)

ditto

Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >S [](start = 48, length = 1)

casing is inconsistent - for job action you did not capitalise enum value descriptions

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >disable - disable the job. This is equivalent to calling the disable job API, with a disableTasks value of requeue. terminate [](start = 61, length = 125)

maybe there is no better option with the present d.m.c/AutoRest situation but having a sentence after one enum value and then going straight into the name of the next is really hard to read.  I don't know if embedded newlines or any other formatting will save us - if not perhaps we need something like:

"none - ....; disable - disables the job; terminate - terminates the job.  The disable option is equivalent to...  In the terminate option, the terminateReason..."

Still gets messy though.  We need them to fix this paragraphing issue!

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >If applied [](start = 422, length = 10)

minor: perhaps "If the job option is applied to a job schedule..."

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >Specifies whether [](start = 20, length = 17)

Is 'specifies' safe here?  I thought we preferred phrases that could be prefixed with "Gets or sets", e.g. "Whether...", but perhaps that's no longer needed?

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >Specifies [](start = 20, length = 9)

Same question about 'specifies'

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >It is common to use a GUID for the id. [](start = 167, length = 38)

remove this

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >Specifies [](start = 20, length = 9)

same comment

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >It is common to use a GUID for the id. [](start = 167, length = 38)

remove this

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >id [](start = 287, length = 2)

probably needs to be capitalised ID

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >ids that can be parsed as integers. For example, a task can depend on task ids 9 to 12, but cannot depend on task ids [](start = 130, length = 117)

capitalise "IDs" now?  (x3)

Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >\"mytask\" to \"yourtask\" [](start = 248, length = 26)

for the future, perhaps, but this is kind of inherent in the type spec.  I think we reworked this in the C# docs to illustrate how an integer range mapped to stringy IDs - perhaps we should review and decide which we prefer.  Not needed for this port anyway

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >Values [](start = 26, length = 6)

the order of these seems pretty random.  Might be good to try to organise them a bit more given the size of the enum (though I'm not sure whether there is an order that people would find predictable, other than plain alphabetical which may not be all that helpful)

Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >Running tasks [](start = 1217, length = 13)

this becomes confusing especially with the flattening of the paragraphs.  Perhaps "Tasks which were running on the node when it was pre-empted..."?

Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >  [](start = 830, length = 1)

needs a dash

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >Note [](start = 26, length = 4)

this was in a title somewhere else - I think it is probably better in the description

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (itowlson) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: >delete [](start = 667, length = 6)

operation?

**Review by itowlson — APPROVED** [review]
:shipit:

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (matthchr) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: Yea, I like your language there, used it.

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (matthchr) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: Fixed

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (matthchr) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: The MSDNified name will go away though in the new world order, so I think it's better to use a name that is more Swagger-like since I think that's going to make it easier to discover across the clients/generated markdown.

Ideally this would be some sort of "link"

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (matthchr) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: There is, but again same as above it will go away when we start generating this content from Swagger I think.

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (matthchr) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: I added newlines here as per @dsgouda's feedback

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (matthchr) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: It is not safe here, fixed.

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (matthchr) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: Yeah, fair point

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (matthchr) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: They are described in the order they are listed in the enum in the spec (so from 0 to N). I think that order makes the most sense, and I can't change that order because it's technically a breaking change.

Comment (matthchr) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: Eyes like a hawk

File: specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json
Comment (matthchr) [inline specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json]: I can't find any places where this is in a title anymore

**Comment by matthchr** [comment]
@dsgouda If the linter looks good you can feel free to merge this now

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**
